compute/k8s-services/vars.tf

```diff
@@ -704,6 +704,15 @@ variable "crossplane_confluent_clusters_endpoints" {
 # Traefik v2 through Flux CD
 # --------------------------------------------------
 
+# Using the variant variables one can perform a blue/green update on Traefik,
+# routing traffic gradually to a new version and then decomissioning an older
+# version without downtime.
+
+# TODO(emil): Rename the original Traefik instance resources and variables to
+# specify that they refer to the "blue" variant after the "blue" instance is
+# destroyed.  This is to avoid downtime or having to reimport resources due to
+# renaming.
+
 variable "traefik_flux_github_owner" {
   type        = string
   description = "Name of the Treaefik Flux repo Github owner (previously: organization)"
```

Comment:
Typo. Not actually introduced by you, but please fix since it is the same file as you are already chaning.

'Traefik' not 'Treaefik'

Reply:
Fixed.